ex7.5_NumPy__vector_matrix.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def create_vector_to_matrix(matrix: np, vector: np):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/